lcls-twincat-pmps/PMPS/PMPS.plcproj

```diff
@@ -483,22 +510,22 @@
   </ItemGroup>
   <ItemGroup>
     <PlaceholderResolution Include="LCLS General">
-      <Resolution>LCLS General, * (SLAC)</Resolution>
+      <Resolution>LCLS General, 0.0.0 (SLAC)</Resolution>
```

Comment:
Do we need to release general lib and switch this back?

Reply:
I think this is the only thing that really needs fixing in the PR

Reply:
Thanks for all your comments Zach. I will work on addressing them soon before we start with the KFE deployment.